gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SortGradeItemsByCategoryPanel.html

```diff
@@ -4,32 +4,42 @@
 <body>
 	<wicket:panel>
 
-		<div id="gradebookSorter" class="by-category" role="list" aria-dropeffect="move">
-			<span class="sr-only" wicket:message="aria-label:sortgradeitems.instructions">Instructions for sorting will go here.</span>
+	<div id="gradebookSorter" class="by-category">
+      <span id="sortInstr" class="sr-only">
```

Comment:
Imo - and I know it's not much of a biggy - we should be sticking to kebab case for tag and attribute names. It's the most widely used convention.